modules/images/webp-uploads/load.php

```diff
@@ -573,3 +573,106 @@ function webp_uploads_update_featured_image( $html, $post_id, $attachment_id ) {
 	return webp_uploads_img_tag_update_mime_type( $html, 'post_thumbnail_html', $attachment_id );
 }
 add_filter( 'post_thumbnail_html', 'webp_uploads_update_featured_image', 10, 3 );
+
+/**
+ * Adds a fallback mechanism to replace webp images with jpeg alternatives on older browsers.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_wepb_fallback() {
+	// Get mime type transofrms for the site.
+	$transforms = webp_uploads_get_upload_image_mime_transforms();
+	if ( ! is_array( $transforms ) ) {
+		return;
+	}
+
+	// We need to add fallback only if jpeg alternatives for the webp images are enabled for the server.
+	$jpegs = in_array( 'image/jpeg', $transforms['image/jpeg'], true ) && in_array( 'image/webp', $transforms['image/jpeg'], true );
+	$webp  = in_array( 'image/jpeg', $transforms['image/webp'], true );
+	if ( ! $jpegs && ! $webp ) {
+		return;
+	}
+
+	?>
+	<script>
+		( function() {
+			window.fallbackWebpImages = function( media ) {
```

Comment:
Probably we want to use 
```suggestion
			window._fallbackWebpImages = function( media ) {
```

To avoid any possible conflict, another thing is  to make sure the object has not been defined already such as:

```suggestion
	if ( '_ fallbackWebpImages' in window && typeof window['fallbackWebpImages'] === 'function') { 
	return;
	}
```

Reply:
Let's prefix this global function with something specific to this module, e.g. `webpUploadsFallbackWebpImages`.

Reply:
Renamed.